<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EchoApp</class>
 <widget class="QWidget" name="EchoApp">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>304</width>
    <height>253</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>echo</string>
  </property>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>301</width>
     <height>251</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QLineEdit" name="inputName">
      <property name="text">
       <string>jason</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="inputMessage"/>
    </item>
    <item>
     <widget class="QPushButton" name="sendButton">
      <property name="text">
       <string>Send</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QTextBrowser" name="dialogWindow"/>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
